podcast/2.18.md

```diff
@@ -0,0 +1,545 @@
+
+**Beyang Liu:**:
+
+Totally. So we are coming up on the end of time here, but I was wondering as, the last parting thought, do you have any thoughts or maybe prognostications that you'd like to leave listeners with about the future of developer tools, or web development, or maybe just the path forward at Mailchimp?
+
+**Kelly Norton:**
+
+Yeah, I mean, so developer tools are really interesting because I would just say that … I don't know if this is a forward-looking statement or not, but I when we were doing Google Web Toolkit, and when we did things like Speed Tracer, and we really focused on developer tools, and then we left Google intent on starting a business, we intentionally avoided developer tools because we felt that having worked in that space, that it was a thing that nobody paid for.
+
+I look around these days and I am just ecstatic that Sourcegraph exists. LaunchDarkly exists. Many of these tools that are things you would build yourself and never possibly outsource are now businesses. And I think while having these things be totally separate does introduce some complexity into our lives, building a software business, having the software industry make a pivot where they are willing to actually pay and recognize the value of developer tools is really exciting.
+
+So in general, I'm just ecstatic to see that there are developer tool businesses that are doing very well. And so, I don't know, I now wonder if the market had been like it is now when we left Google, intent on starting a business, would we have been more eager to start a dev tools business since that was our passion? I mean, I love what we created, but boy did it take us a long time to get to that, largely because we were operating in spaces we didn't understand, whereas we absolutely did understand developer tools and loved it.
+
+**Beyang Liu:**:
+
+Yeah, the world has changed a lot. I remember we started Sourcegraph back in 2013. It was very different than ... You'd say the words ‘developer tool’ to an investor, and they'd almost get up out of the seat and like run away because the consensus was there wasn't a viable business strategy for it. But now, it's completely different, so many awesome companies, a lot of which we feature on this podcast, fortunately enough. And it's just great to see the innovation that's advancing developer productivity and spreading it across all the companies building software today.
```

Comment:
```suggestion
Yeah, the world has changed a lot. I remember we started Sourcegraph back in 2013. It was very different then ... You'd say the words ‘developer tool’ to an investor, and they'd almost get up out of the seat and like run away because the consensus was there wasn't a viable business strategy for it. But now, it's completely different, so many awesome companies, a lot of which we feature on this podcast, fortunately enough. And it's just great to see the innovation that's advancing developer productivity and spreading it across all the companies building software today.
```